.eslintrc.json

```diff
@@ -24,7 +24,7 @@
         "curly": "off",
         "default-case": "off",
         "dot-location": "off",
-        "dot-notation": "off",
+        "dot-notation": "error",

```

Comment:
@cameronwolf should it be "warn" instead of "error"?  Error is maybe overly strict for most of these.